password/password.cabal

```
@@ -56,6 +56,11 @@ flag bcrypt
   default: True
   manual: True
 
+flag crypton
+  description: Compile with Scrypt support?
+  default: False
+  manual: True
+
```

I thought it might be nice to go ahead and add the `cryptonite` flag in right now, in case there are users that want to set this in advance of the switch.

```suggestion

flag cryptonite
  description: Use the cryptonite library as the cryptographic backend
  default: True
  manual: True

```

Also, I guess we should have some sort of assertion that exactly one of the `crypton` / `cryptonite` flags is set.  I'm not sure how to do this off the top of my head though.

---

I feel it's a good idea to include the flag, but I'd argue to add to the `cryptonite` flag description: "... (does nothing until `crypton` becomes the default)"

And then when we switch in a major version, that part is removed and the `crypton` flag description turns into "This flag is here for backwards compatibility"

---

Yeah, that sounds good!